test/test_torch.py

```diff
@@ -4773,6 +4773,9 @@ def test_logspace(self):
         self.assertRaises(RuntimeError, lambda: torch.logspace(0, 1, -1))
         self.assertEqual(torch.logspace(0, 1, 1), torch.ones(1), 0)
 
+        # Check non-default base=2
```

Comment:
could you find the similar test in `test_cuda` and add a similar assertion there?

Reply:
done in test/test_cuda.py

Reply:
thanks. I noticed that this test assertion is just a really simple case where `start == end`.. could you add an `assertEqual` for something like `torch.logspace(0, 2, 3, base=2)`?

Reply:
I added one more assertion. I didn't add more because I thought all the previous tests for default base=10 were enough.
Hope that's OK now.